pep8.py

```diff
@@ -54,11 +54,13 @@
 import inspect
 import keyword
 import tokenize
+import codecs
+import io
```

Comment:
Just noticed this isn't available in Python 2.5, which seems to still be supported (based on [the changelog](https://pypi.python.org/pypi/pep8)). 

Is this still the case? I'll rework if so.